extensions/amp-analytics/0.1/vendors.js

```diff
@@ -41,25 +41,30 @@ export const ANALYTICS_CONFIG = {
       'scrollHeight': 'SCROLL_HEIGHT',
       'screenWidth': 'SCREEN_WIDTH',
       'screenHeight': 'SCREEN_HEIGHT'
+      // Vars that GA can use: documentEncoding, userLanguage
```

Comment:
Lets add it to #1503? I'll remove this line from here once the issue is updated or a new one is created.